docs/guide-zh-CN/input-validation.md

```diff
@@ -334,7 +334,7 @@ class CountryValidator extends Validator
     public function validateAttribute($model, $attribute)
     {
         if (!in_array($model->$attribute, ['兲朝', '墙外'])) {
-            $this->addError($attribute, '国家必须为 "兲朝" 或 "墙外" 中的一个。');
+            $model->addError($attribute, '国家必须为 "兲朝" 或 "墙外" 中的一个。');
```

Comment:
Seems it's outdated. See original: https://github.com/yiisoft/yii2/blob/master/docs/guide/input-validation.md#standalone-validators-